daemon/src/model/cfd.rs

```diff
@@ -364,8 +364,12 @@ pub enum CfdEvent {
     OracleAttestedPriorCetTimelock {
         #[serde(with = "hex_transaction")]
         timelocked_cet: Transaction,
-        #[serde(with = "hex_transaction")]
-        commit_tx: Transaction,
+        /// The commit transaction for the DLC of this CFD.
+        ///
+        /// If this is set to `Some`, we haven't previously attempted broadcast `commit_tx` and
+        /// need to broadcast it as a result of this event.
+        #[serde(with = "hex_transaction::opt")]
```

Comment:
🤔 I'm surprised that we need a custom implementation here, but you know all the `serde` tricks.